docs/tools/aleph-client/documentation.md

```diff
@@ -0,0 +1,942 @@
+# Aleph CLI
+
+**Usage**:
+
+```console
+$ aleph aleph [OPTIONS] COMMAND [ARGS]...
+```
+
+**Options**:
+
+* `--install-completion`: Install completion for the current shell.
+* `--show-completion`: Show completion for the current shell, to copy it or customize the installation.
+* `--help`: Show this message and exit.
+
+**Commands**:
+
+* `about`: Display the informations of Aleph CLI
+* `account`: Manage account
+* `aggregate`: Manage aggregate messages on aleph.im
+* `domain`: Manage custom Domain (dns) on aleph.im
+* `file`: File uploading and pinning on IPFS and...
+* `instance`: Manage instances (VMs) on aleph.im network
+* `message`: Post, amend, watch and forget messages on...
+* `node`: Get node info on aleph.im network
+* `program`: Upload and update programs on aleph.im VM
+
+## `about`
+
+Display the informations of Aleph CLI
+
+**Usage**:
+
+```console
+$ aleph about [OPTIONS] COMMAND [ARGS]...
+```
+
+**Options**:
+
+* `--help`: Show this message and exit.
+
+**Commands**:
+
+* `version`
+
+### `about version`
+
+**Usage**:
+
+```console
+$ aleph about version [OPTIONS]
+```
+
+**Options**:
+
+* `--help`: Show this message and exit.
+
+## `account`
+
+Manage account
+
+**Usage**:
+
+```console
+$ aleph account [OPTIONS] COMMAND [ARGS]...
+```
+
+**Options**:
+
+* `--help`: Show this message and exit.
+
+**Commands**:
+
+* `address`: Display your public address.
+* `balance`
+* `create`: Create or import a private key.
+* `export-private-key`: Display your private key.
+* `path`
+* `sign-bytes`: Sign a message using your private key.
+
+### `account address`
+
+Display your public address.
+
+**Usage**:
+
+```console
+$ aleph account address [OPTIONS]
+```
+
+**Options**:
+
+* `--private-key TEXT`: Your private key. Cannot be used with --private-key-file
+* `--private-key-file PATH`: Path to your private key file  [default: /home/mhh/.aleph-im/private-keys/ethereum.key]
+* `--help`: Show this message and exit.
+
+### `account balance`
+
+**Usage**:
+
+```console
+$ aleph account balance [OPTIONS]
+```
+
+**Options**:
+
+* `--address TEXT`: Address
+* `--private-key TEXT`: Your private key. Cannot be used with --private-key-file
+* `--private-key-file PATH`: Path to your private key file  [default: /home/mhh/.aleph-im/private-keys/ethereum.key]
+* `--help`: Show this message and exit.
+
+### `account create`
+
+Create or import a private key.
+
+**Usage**:
+
+```console
+$ aleph account create [OPTIONS]
+```
+
+**Options**:
+
+* `--private-key TEXT`: Your private key. Cannot be used with --private-key-file
```

Comment:
The explanation of options is good but I think that later we could make it better looking by making a tab like this: 
<img width="718" alt="image" src="https://github.com/aleph-im/aleph-docs/assets/91880456/51b80853-bd70-4a7d-923c-920972ade269">

Reply:
Agree, in order to make this more automated, I might write some additional script for reformatting the output of the typer docs generator

Reply:
It's a drudge currently to do it manually ^^'